.github/workflows/deploy-storybook.yml

```diff
@@ -0,0 +1,24 @@
+name: Deploy Storybook
+on:
+  push:
+    branches: [main]
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  deploy:
+    name: Build & deploy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Determine node version
+        run:
+          echo "NODE_VERSION=$(grep -oP 'node\K\d+' Dockerfile)" >> $GITHUB_ENV
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '${{ env.NODE_VERSION }}'
+          cache: yarn
+      - run: yarn install --frozen--lockfile
+      - run: yarn deploy-storybook --ci
```

Comment:
@jgerigmeyer The original job wrapped this in an if:

```bash
if [ -n "${DEPLOY_STORYBOOK}" ]; then
   yarn deploy-storybook --ci
fi
```
Is that necessary? I couldn't find where that env var was set

Reply:
Yeah, that was set directly on our CircleCI, with the goal of this step being skipped on SFDO's CircleCI runs without officially failing the CI check. Let's hold off on that for now, until we have a clearer sense of the eventual hand-off path here.